tests/Feature/Framework/ResolveDifferentProjectNamespaces/FeatureTest.php

```diff
@@ -22,7 +23,7 @@ final class FeatureTest extends TestCase
     public function setUp(): void
     {
         Gacela::bootstrap(__DIR__, static function (GacelaConfig $config): void {
-            $config->addAppConfig('config/default.php');
+            $config->addAppConfigKeyValue(GacelaCache::KEY_ENABLED, false);
```

Comment:
Not sure if I understand the PR.

```php
# default.php
return [
    GacelaCache::KEY_ENABLED => false,
];

// is the same than?

# index.php
Gacela::bootstrap(__DIR__, static function (GacelaConfig $config): void {
    $config->addAppConfigKeyValue(GacelaCache::KEY_ENABLED, false);
}
```
If so, in the `addAppConfigKeyValue()` method, the `GacelaCache::KEY_ENABLED` is the key and the `false` is the value?

Then, how could I add multiple keys? Like this? 🤔 
```php
$config->addAppConfigKeyValue(GacelaCache::KEY_ENABLED, false);
$config->addAppConfigKeyValue('some key', 'some value');
$config->addAppConfigKeyValue('other key', 'other value');
```

What about this proposal @Chemaclass? I would find even more interested to do something like

```php
$config->addAppConfigKeyValue([
    GacelaCache::KEY_ENABLED => false,
    'some key' => 'some value',
    'other key' => 'other value',
]);
```

Reply:
Yes, that would be another way. Do you want to do it in a follow up PR, @JesusValera?

Reply:
Sure thing :)